src/foam/dao/CSVSink.js

```diff
@@ -68,14 +68,19 @@ foam.CLASS({
           .setProps(getProps())
           .build();
       `
-    }
+    },
```

Comment:
please see CSVTableExportDriver, which is alternative for current CSVDriver. the difference between CSVTableExportDriver and CSVDriver is that CSVTableExportDriver uses Projection ( which returns array of values for provided properties ) while CSVDriver uses CSVSink which retrieves values using CSVOutputterImpl which calls property's toCSV method on objects, which are returned on DAO

so please take a look at CSVTableExportDriver, and let me know which you prefer
thanks